lib/cheffish/merged_config.rb

```diff
@@ -24,7 +24,7 @@ def [](name)
       else
         result_configs = []
         configs.each do |config|
-          value = config[name]
+          value = config[name.to_s] || config[name.to_sym]
```

Comment:
I also am concerned think this will cause the value to never be equal to `nil`, since it would convert the value of `nil` to a empty string.

Eg:

``` Ruby
[1] pry(main)> (nil).nil?
=> true
[2] pry(main)> (nil.to_s).nil?
=> false
```